baseapp/baseapp.go

```diff
@@ -535,19 +537,25 @@ func (app *BaseApp) runTx(mode runTxMode, txBytes []byte, tx sdk.Tx) (result sdk
 		if r := recover(); r != nil {
 			switch rType := r.(type) {
 			case sdk.ErrorOutOfGas:
-				log := fmt.Sprintf(
-					"out of gas in location: %v; gasWanted: %d, gasUsed: %d",
-					rType.Descriptor, gasWanted, ctx.GasMeter().GasConsumed(),
+				err = sdkerrors.Wrap(
```

Comment:
If `sdk.ErrorOutOfGas` were an `sdkerrors` type, then we would not even need to cast it here. Just `err = rType`.

Both for consistency/aesthertics, but also one thing lost here is the stack trace. The original error that was thrown in panic would have the original stack trace. Maybe this is a minor point, but maybe interesting for development and likely minimal work to change.

Reply:
Added a TODO to address in a subsequent PR.